### ISSUE TITLE
getPlayerInfo

### ISSUE THREAD

**Issue by Zlecktov** [sentanos/roblox-js#56]
One function to rule them all.   Just a function that combines getting the Blurb, Status, Username, Age, etc. all into one function and returns the data in an array.

**Comment by sentanos** [comment]
Yeah after the addition of another player profile function I going to add this. Unfortunately I have not had time lately to work on roblox-js but this will be added to the next version.

**Comment by Neztore** [comment]
@sentanos @Zlecktov would it be better to have it returned in an object or an array? An object would make it easier to access data.
